services/ingest.json

```diff
@@ -29,55 +47,55 @@
     {
       "groupId": "io.jenkins.blueocean",
       "artifactId": "blueocean",
-      "url": "http://updates.jenkins-ci.org/download/plugins/blueocean/1.6.2/blueocean.hpi",
+      "url": "http://updates.jenkins-ci.org/download/plugins/blueocean/1.7.0/blueocean.hpi",
```

Comment:
As noted in Gitter, so long as transitive dependencies are omitted from `essentials.yaml` and their versions picked up from the UC, your manifest will be **nondeterministic**, which is terrible. Best to force the editor of `essentials.yaml` to specify every plugin to be included and its version, and fail the `make` target if there is any inconsistency (missing plugin, plugin too old, plugin incompatible with Jenkins core, etc.). Failing that, at least use the lowest upper bound of the dependency versions requested in the graph, instead of looking in `update-center.json`, so that the resulting version is deterministic.